xbmc/interfaces/legacy/ModuleXbmcplugin.h

```diff
@@ -255,7 +266,7 @@ namespace XBMCAddon
     ///
     addSortMethod(...);
 #else
-    void addSortMethod(int handle, int sortMethod, const String& label2Mask = emptyString);
+    void addSortMethod(int handle, int sortMethod, const String& label2Mask = emptyString, const String& labelMask = emptyString);
```

Comment:
Since v19 is a breaking change for the python API probably it makes sense to reorder the arguments. Or take this is consideration in https://github.com/xbmc/xbmc/pull/17456

Reply:
yup, this is a typical example of the limitations of always trying to be backward compatible.
new options will have to be added in a clumsy / non-logical manner just to ensure we do not break anything.